src/types/styled-components.d.ts

```diff
@@ -1,9 +1,10 @@
-import { CSSProp } from "styled-components"
+import { CSSProp } from 'styled-components'
 
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
```

Comment:
I had to disable this eslint rule since by default typescript-eslint doesn't allow to declare empty interfaces.